cirq/ops/raw_types.py

```diff
@@ -449,6 +449,14 @@ def _decompose_(self, qubits):
         return protocols.inverse(
             protocols.decompose_once_with_qubits(self._original, qubits))
 
+    def _has_unitary_(self):
+        from cirq import protocols, devices
+        qubits = devices.LineQubit.range(self._num_qubits_())
```

Comment:
This should be `LineQid.for_gate(self)`.

Reply:
Done.